code/ch02/op_as.dart

```diff
@@ -20,7 +20,8 @@ main() {
   // Prints:
   // in Person
   // in Employee
-  if (emp is Person) { // Type check
```

Comment:
Why did the comment change lines? It's line 23 that's the type check, but moving it implies that line 25 might be it...